### ISSUE TITLE
1.16.3 - Ice Dragons in caves all male....Ice Dragons in roosts all Female?

### ISSUE THREAD

**Issue by Dwarfy-Dug** [Alex-the-666/Ice_and_Fire#3036]
I know this could be random number generation causing a definite bias...but to check I swapped to creative mode, emplyed the X-Ray mod....and went checking every cave I could find in a nearby snowy biome. Every cave dragon I found was male. 8 located and checked. 
Saw quite a few Ice Dragons on the surface roosts....all were Female.

**Comment by Dwarfy-Dug** [comment]
OK...additional info..... for my games, the dragon 'gender' is definitely NOT random, and seems very broken. 
Following on from the above...I created another world at random, spawned in...and repeated the process starting with fire dragons and going from there. This time...every fire dragon in a cave was male...

Now....in my initial game, it was already in existence when the most recent patch, with the ghosts and graveyards was applied. I only went to the Ice biome after the patch was released. Ever since that patch...I have had no female cave dragons spawn. My fresh game confirmed it was not a dragon type thing....it was just the chunks had already been created and the dragon sex fixed before the patch.

Obviously...this is a bit of a game breaker....
